### PR TITLE
Story/graph sidebar/492

### PR DESCRIPTION
**Pull request by Kyubinhan**
* Add a resizable sidebar to the graph to control the window of the x domain.
* Modify d3 helper functions to return a function that allows redrawing graphics.
* Remove the time range select input since it's no longer needed.

### CODE REVIEW

File: src/features/fireWeather/components/graphs/TempRHGraph.styles.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const modelSummaryRHAreaColor = '#94ffeb'
 export const highResModelTempColor = '#a017c2'
 export const highResModelRHColor = '#3ac417'
 export const highResModelSummaryTempAreaColor = '#ddbae9'
-export const highResModelSummaryRHAreaColor = '#6bbbf5'
+export const highResModelSummaryRHAreaColor = '#b5f0a5'
```
Comment (Sybrand) [inline src/features/fireWeather/components/graphs/TempRHGraph.styles.ts:15]: Good catch matching the colours. You probably want to check for colourblindness issues again now that it's using the correct colour.

File: src/utils/storybook.ts [hunk 2]
```diff
@@ -14,8 +14,8 @@ const getFutureValues = () => {
 
   while (last.diff(first, 'days') >= 0) {
     for (let length = 0; length < 24; length++) {
-      const temp = 15 + Math.random() * 10
-      const rh = 40 + Math.random() * 15
+      const temp = 20 + Math.random() * 10
```
Comment (Sybrand) [inline src/utils/storybook.ts:17]: Might want to stick some negative numbers in? You could use a sine wave to get something that matches reality closer. So like 
```
const temp = Math.sin(i)*20 + Math.random()*10
```

**Review by Sybrand — CHANGES_REQUESTED** [review]
As discussed, fix the x-axis overflow issue and switch to "Oct 3" style dates.
Using sine waves in storybook to generate weather values will give you something that looks more like the diurnal cycle as opposed to a random walk.

File: src/features/fireWeather/components/graphs/TempRHGraph.styles.ts (hunk 1 above)
Comment (Kyubinhan) [reply src/features/fireWeather/components/graphs/TempRHGraph.styles.ts:15]: Checked and done

File: src/utils/storybook.ts (hunk 2 above)
Comment (Kyubinhan) [reply src/utils/storybook.ts]: Done

**Review by Sybrand — COMMENTED** [review]
Looks good apart from a few errors - let me know when they're fixed and I'll approve.

File: src/utils/storybook.ts [hunk 3]
```diff
@@ -170,6 +111,75 @@ const getPastValues = () => {
...
+
+  const days = 2
+  const first = moment()
+    .utc()
+    .set({ minute: 0, second: 0 })
+  const last = moment(first).add(days, 'days')
+
+  while (last.diff(first, 'days') >= 0) {
+    for (let length = 0; length < 24; length++) {
+      const signWeight = 7
```
[the remark this answers is not in the document]
Comment (Kyubinhan) [reply src/utils/storybook.ts:127]: Yes! nice catch

**Review by andrea-williams — APPROVED** [review]
Aside from the Storybook compilation error when running (as Sy mentioned), it looks good to me!

**Review by Sybrand — APPROVED** [review]

**Pull request merged**